* 2019 survey paper: [A benchmark study of k-mer counting methods for high-throughput sequencing](https://academic.oup.com/gigascience/article/7/12/giy125/5140149)
* 2021 ht kmer: [CHTKC: a robust and efficient k-mer counting algorithm based on a lock-free chaining hash table](https://academic.oup.com/bib/article/22/3/bbaa063/5841329)
* 2017 survey: [Next-generation sequencing: big datameets high performance computing](https://reader.elsevier.com/reader/sd/pii/S1359644617300582)
* [Distributed-Memory k-mer Counting on GPUs](https://ieeexplore.ieee.org/abstract/document/9460480)
* [NEST: DIMM based Near-Data-Processing Accelerator for K-mer Counting](https://ieeexplore.ieee.org/abstract/document/9256651)
* [Optimizing High Performance Distributed Memory Parallel Hash Tables for DNA k-mer Counting](https://ieeexplore.ieee.org/abstract/document/8665746)

* Jellyfish
    * [Homepage](https://www.psc.edu/user-resources/software/jellyfish)
    * [V1 paper](https://academic.oup.com/bioinformatics/article/27/6/764/234905)
    * [V2 user guide](http://www.genome.umd.edu/docs/JellyfishUserGuide.pdf)

* [Non-blocking hashtables with open addressing](https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-639.pdf) (Jellyfish hashtable)

* [Performance extraction and suitability analysis of multi- and many-core architectures for next generation sequencing secondary analysis](https://dl.acm.org/doi/abs/10.1145/3243176.3243197?casa_token=5BhKH0oWtQEAAAAA:NY76jrngG5ZOl9CNREBx7j11v0wD_T1DYEqcXtzlOCgaoROaqM5fNFdh_VsTG_rNRD2-169vbGL0mQ) -- Uses Kemerind for alignment on Xeon and Xeon Phi
